libmamba/src/api/configuration.cpp

```diff
@@ -490,6 +490,18 @@ namespace mamba
             {
                 for (const auto& dir : dirs)
                 {
+                    if (!fs::exists(dir))
+                    {
+                        try
+                        {
+                            fs::create_directories(dir);
+                        }
+                        catch (const fs::filesystem_error& e)
+                        {
+                            LOG_WARNING << "Error creating directory " << dir << ": " << e.what()
```

Comment:
Do we really want to continue when an error occurs here?
Did you consider not catching the error so that that we stop immediately and let the caller handle the issue?

Reply:
Yes, this is intentional.  I wanted to preserve the same behavior as mamba 1, which continues if it cannot create the directory.